src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -392,6 +457,19 @@ public native Object[] newSpecialArray(Class<?> componentType,
 
     /**
      * Exits the larval state and returns a value instance.
+     * <p>
+     * This method mimics the action of passing a larval object to a do-nothing
+     * constructor to obtain a non-larval object. This method exhibits
+     * {@linkplain Unsafe undefined behavior} unless all the following
+     * conditions are true:
+     * <ul>
+     * <li>The argument must be a larval value object created by
+     * {@link #makePrivateBuffer(Object)}.
+     * <li>After the invocation of this method, the variable that holds the
+     * argument passed into this method must not be used.
+     * </ul>
```

Comment:
I would add the exhortation that every field of the value object must have a value.